src/net/bus.c

```diff
@@ -1194,8 +1240,12 @@ void BUS_init(Bus* bus, int num_periods) {
 
   bus->index = 0;
 
-  bus->v_max = BUS_DEFAULT_V_MAX;
-  bus->v_min = BUS_DEFAULT_V_MIN;
+  bus->v_max_reg = BUS_DEFAULT_V_MAX_REG;
+  bus->v_min_reg = BUS_DEFAULT_V_MIN_REG;
+  bus->v_max_norm = BUS_DEFAULT_V_MAX_LIMIT;
+  bus->v_min_norm = BUS_DEFAULT_V_MIN_LIMIT;
+  bus->v_max_emer = BUS_DEFAULT_V_MAX_LIMIT;
+  bus->v_min_emer = BUS_DEFAULT_V_MIN_LIMIT;
```

Comment:
As in a previous comment, we should have only BUS_DEFAULT_V_MAX/MIN. No need for more.